PostProc/Data_Extraction/data_extraction.py

```diff
@@ -29,7 +30,45 @@ def load_4d_emep_data(ds, t, varName, outArray):
     varData = ds[varName][t, :, :, :]
     outArray[t, :, :, :] = to_np(varData)
 
-#def load_4d_emep_PM25_data(ds, t, outArray):
+def load_4d_emep_nox(ds, t, outArray):
+    no = ds["NO"][t, :, :, :]
+    no2 = ds["NO2"][t, :, :, :]
+    outArray[t, :, :, :] = to_np(no) + to_np(no2) # NOX = NO + NO2
+
+def load_4d_emep_pm25(ds, t, outArray):
+    kg_air_per_mol = 0.0289647  # mean molecular weight of dry air in kg/mol
+    air_density = 1.1845 # at 1 atm and 298K (kg/m3)
+
+    pmfine_vars = [
+        "SO4", "NO3_f", "NH4_f", "EC_f_wood_new", "EC_f_wood_age",
+        "EC_f_ffuel_new", "EC_f_ffuel_age", "pSO4f", "remPPM25", "OM25_p",
+        "VBS_TEST", "Ash_f", "ffire_BC", "ffire_remPPM25", "SeaSalt_f",
+        "Dust_road_f", "Dust_wb_f", "Dust_sah_f"
+    ]
+
+    pmfine_mw = {
+        "SO4": 96, "NO3_f": 62, "NH4_f": 18, "EC_f_wood_new": 12, "EC_f_wood_age": 12,
+        "EC_f_ffuel_new": 12, "EC_f_ffuel_age": 12, "pSO4f": 96, "remPPM25": 12, "OM25_p": 1,
+        "VBS_TEST": 1, "Ash_f": 12, "ffire_BC": 12, "ffire_remPPM25": 12, "SeaSalt_f": 58,
+        "Dust_road_f": 200, "Dust_wb_f": 200, "Dust_sah_f": 200, "NO3_c": 62
+    }
+    
+    # Y = X * (MWspec / MWair)
+    # Y: mass fraction (kg/kg), X: mix ratio (mol/mol), MW: molecular weight (kg/mol)
+
+    pm25 = np.zeros_like(ds["SO4"][t, :, :, :])
+    for var in pmfine_vars:
+        mw = pmfine_mw[var] / 1000.0
```

Comment:
can you replace the bare 1000 with a variable that's named for what the conversion is (eg. kg_to_g)